### PR TITLE
Updates to docs and imports for classic.py

### PR DESCRIPTION
**Pull request by dschult**
Minimal document updates... it snowballed from what I intended.
I don't think it conflicts with #4401

### CODE REVIEW

File: networkx/generators/classic.py [hunk 1]
```diff
@@ -66,10 +62,10 @@ def _tree_edges(n, r):
 
 
 def full_rary_tree(r, n, create_using=None):
-    """Creates a full r-ary tree of n vertices.
+    """Creates a full r-ary tree of n nodes.
```
Comment (rossbar) [inline networkx/generators/classic.py:65]:
```suggestion
    """Creates a full r-ary tree of `n` nodes.
```

File: networkx/generators/classic.py [hunk 2]
```diff
@@ -1,23 +1,19 @@
 """Generators for some classic graphs.
 
-The typical graph generator is called as follows:
+The typical graph builder function is called as follows:
 
 >>> G = nx.complete_graph(100)
 
 returning the complete graph on n nodes labeled 0, .., 99
-as a simple graph. Except for empty_graph, all the generators
+as a simple graph. Except for empty_graph, all the functions
```
Comment (rossbar) [inline networkx/generators/classic.py:8]:
```suggestion
as a simple graph. Except for `empty_graph`, all the functions
```
Might as well make it a link in the html

**Review by rossbar — APPROVED** [review]

**Comment by jarrodmillman** [comment]
See https://github.com/dschult/networkx/pull/13.

**Comment by jarrodmillman** [comment]
I think we should avoid changing imports in this PR.  I am not sure how we should do the imports and I think we need to do more research.  The discussion on https://github.com/dschult/networkx/pull/13 is getting long (mostly b/c of me).  The other changes here seem good and should land quickly.

We should have a separate PR (perhaps a brief NXEP even) sorting out how we do imports.  In particular, we need to figure out how to stop our circular imports; currently
```
import networkx
```
gives us
```
networkx
networkx.networkx
networkx.nx
...
networkx.nx.networkx.nx
...
```

And
```
import networkx as nx
```
gives us
```
nx
nx.networkx
nx.nx
...
nx.nx.networkx.nx
...
```


We should do some experiments to see exactly what gets imported in what order, when.

Also, whether that changes when you use absolute vs relative imports.

**Comment by rossbar** [comment]
I think @jarrodmillman 's suggestion is a good one - my approval stands for the doc changes.

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**